lib/OpenQA/Git.pm

```diff
@@ -171,4 +176,35 @@ sub is_workdir_clean ($self) {
     return $r->{status};
 }
 
+sub cache_ref ($self, $ref, $url, $relative_path, $output_file, $allow_arbitrary_url_fetch = 1) {
+    $self->app->log->debug(
```

Comment:
I just realized that the debugging is a bit too noisy in case the file is already there. Maybe this could be moved below line 187

Reply:
https://github.com/os-autoinst/openQA/pull/6226